src/field_theory/normal.lean

```diff
@@ -68,4 +68,25 @@ begin
     (minimal_polynomial.dvd_map_of_is_scalar_tower K hx)⟩,
 end
 
+variables {F} {E} {E' : Type*} [field E'] [algebra F E']
+
+lemma normal.of_alg_equiv (h : normal F E) (f : E ≃ₐ[F] E') : normal F E' :=
```

Comment:
```suggestion
lemma normal.of_alg_equiv [h : normal F E] (f : E ≃ₐ[F] E') : normal F E' :=
```
you've declared `normal` to be a typeclass, so I think it makes sense to use it that way.
(I think that in a previous review I missed the same issue with `normal.tower_top_of_normal`.)